src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -173,167 +222,175 @@ <h2>Request for judgment by default</h2>
         </td>
       </tr>
       <tr>
-        <td>Amount already paid:</td>
+        <td><strong>Date interest is claimed from:</strong></td>
         <td>
-          {{ ccj.amount.paidAmount }}
+          {{ ccj.amount.interest.fromDate }}
         </td>
       </tr>
       <tr>
-        <td>Amount to be paid by defendant:</td>
+        <td><strong>Interest daily amount:</strong></td>
         <td>
-          {{ ccj.amount.remainingAmount }}
+          {{ ccj.amount.interest.dailyAmount }}
         </td>
       </tr>
 
       {% if ccj.repaymentPlan.repaymentOption == "Immediately" %}
         <tr>
-          <td>When you want the defendant to pay:</td>
+          <td><strong>When you want the defendant to pay:</strong></td>
           <td>
             {{ ccj.repaymentPlan.repaymentOption }}
           </td>
         </tr>
       {% else %}
         <tr>
-          <td>How you want the defendant to pay:</td>
+          <td><strong>How you want the defendant to pay:</strong></td>
           <td>
             {{ ccj.repaymentPlan.repaymentOption }}
           </td>
         </tr>
         {% if ccj.repaymentPlan.repaymentOption == "By a set date" %}
           <tr class="no-border">
-            <td>When you want the defendant to pay:</td>
+            <td><strong>When you want the defendant to pay:</strong></td>
             <td>
               {{ ccj.repaymentPlan.paySetByDate }}
             </td>
           </tr>
         {% endif %}
         {% if ccj.repaymentPlan.repaymentOption == "By instalments" %}
           <tr class="no-border">
-            <td>Instalments of:</td>
+            <td><strong>Instalments of:</strong></td>
             <td>
               {{ ccj.repaymentPlan.instalmentAmount }}
             </td>
           </tr>
           <tr class="no-border">
-            <td>When you want them to make the first payment:</td>
+            <td><strong>When you want them to make the first payment:</strong></td>
             <td>
               {{ ccj.repaymentPlan.firstPaymentDate}}
             </td>
           </tr>
           <tr class="no-border">
-            <td>How often you want them to make this payment:</td>
+            <td><strong>How often you want them to make this payment:</strong></td>
             <td>
               {{ ccj.repaymentPlan.paymentSchedule }}
             </td>
           </tr>
         {% endif %}
       {% endif %}
-
     </table>
   </div>
 </div>
-<div class="section">
-  <div class="section-heading">
-    <strong>Declaration</strong>
-  </div>
-  <div class="section-body">
-    <table>
-      {% if ccj.claim.countyCourtJudgment.statementOfTruth is not defined %}
+    <div class="section">
+      <div class="section-heading">
+        <strong>Total amount</strong>
+      </div>
+      <div class="section-body">
+          <table>
         <tr>
+          <td><strong>Claim amount:</strong></td>
           <td>
-            Name:
+            {{ ccj.amount.totalAmount }}
```

Comment:
That does not look right. A `ccj.amount.totalAmount` is used both in `Claim amount:` and `Total:` rows.